specification.md

```diff
@@ -198,6 +198,35 @@ Each Schedule object may optionally include the following extension JSON objects
 	|---|---|---|
 	|`url`| string | fixed value of `"http://fhir-registry.smarthealthit.org/StructureDefinition/vaccine-dose"`|
 	|`valueInteger` | number | indicates sequence number (should be be `1` or `2` for current vaccines)|
+	
+* "Eligibility" extension: used to convey eligibility requirements to book appointments on this Schedule. Each eligibility requirement is effectively a tag, and clients should evaluate each eligibility requirement independently. We do not attempt to support expression trees or other complex logic.
+
+	| field name | type | description |
+	|---|---|---|
+	|`url`| string | fixed value of `"http://fhir-registry.smarthealthit.org/StructureDefinition/eligibility"`|
+	|`valueCoding` | JSON object| Coded representation of an eligibility requirement|
+	| &nbsp;&nbsp;&rarr;&nbsp;`system` | string | See Eligibility codings below |
+	| &nbsp;&nbsp;&rarr;&nbsp;`code` | string |  See Eligibility codings below |
+	| &nbsp;&nbsp;&rarr;&nbsp;`display` | string | See Eligibility codings below |
+	
+  * _Eligibility Requirement: Invitation Required_. Appointments are only available to book after the user has received an invitation (e.g., after making it to the top of  a waiting list).
+  ```
+  {
+    "system": "http://fhir-registry.smarthealthit.org/CodeSystem/eligibility",
+    "code": "invitation-required",
+    "display": "Invitation Required"
+  }
+  ```
```

Comment:
I’m not sure `invitation-required` is clear enough. I’m thinking of two real-world cases that would fit here:
- You have to join a waitlist in order to get the invitation code.
- You were given an invitation code, sight-unseen. California and some other states are doing this as an equitability approach, where households under a certain income threshold may be sent a code to use, or outreach workers may be giving out codes to people in a disadvantaged neighborhood.

This doesn’t tell me whether I can go *get* an invitation somehow (e.g. by signing up for a waitlist).

Reply:
> Yeeesh, I somehow totally missed this PR that’s been sitting for weeks. I am so sorry. 😬
> 
> I’m wondering whether walk-ins allowed / appointments allowed also fit here. We have locations that are:
> 
> * Only walk-ins
> * Only appointments (most common for COVID-19 right now)
> * Walk-ins and appointments (often with different eligibility requirements, e.g. walk-ins or appointments allowed for age >= 50, appointments-only for age < 50)
> 
> Is there a way to do age here, too?

I think getting to business logic like "walk-ins or appointments allowed for age >= 50, appointments-only for age < 50)" is going to be very challenging. These could perhaps be factored out into separate Schedules with "simple rules" for each.